src/lib/proposalUtils.ts

```diff
@@ -1,4 +1,5 @@
 import { copyToClipboard } from "./util";
+import moment = require("moment-timezone");
```

Comment:
how this is related to this pr ?

Reply:
Competition parameters include dates.  This is to support them.